dashboard/test/controllers/foorm/simple_survey_forms_controller_test.rb

```diff
@@ -44,7 +44,7 @@ class SimpleSurveyFormsControllerTest < ActionDispatch::IntegrationTest
 
       get "/form/#{@simple_survey_form.path}/configuration"
       assert_response :success
-      assert_equal 'application/json', @response.content_type
+      assert_equal 'application/json; charset=utf-8', @response.content_type
```

Comment:
We could instead do something like

```suggestion
      assert @response.content_type.starts_with? 'application/json'
```

Depending on exactly what we care about here

Reply:
Thanks for the heads up.  No objection here but @molly-moen or @bencodeorg probably know these tests best.

Reply:
Sounds good to me 👍